model/src/w3profsmd_pdlib.F90

```diff
@@ -610,17 +608,14 @@ SUBROUTINE PDLIB_IOBP_INIT(IMOD)
       END DO
 
       IOBDP_loc = 0
-      !DEALLOCATE(IOBP,IOBPD)
       IOBP => NULL()
       IOBPD => NULL()
       DEALLOCATE(GRIDS(IMOD)%IOBP,GRIDS(IMOD)%IOBPD)
-!
-     CALL SET_IOBPA_PDLIB
-
+      CALL SET_IOBPA_PDLIB
 !/
 !/ End of W3SPR4 ----------------------------------------------------- /
```

Comment:
Should we go ahead and clean up the "end of W3SPR4" here as well?

Reply:
Actually your additions of the subroutine names in the "end" lines have pointed out that we have a lot of "End of XXXX" comments that do not match the subroutines in this file.  Are you interested in cleaning this up now, or leaving this for later?